sunpy/net/jsoc/jsoc.py

```diff
@@ -787,8 +787,11 @@ def _make_recordset(self, series, start_time='', end_time='', wavelength='',
 
         if not pkstr:
             # pkstr cannot be totally empty
-            error_message = "Atleast one PrimeKey must be passed."
-            raise ValueError(error_message)
+            #
+            # Note that whilst it is technically posisble to just search by series,
+            # this is not allowed here, because some of these would be very large
+            # searches that would make JSOC sad
+            raise ValueError("At least one PrimeKey, time, or wavelength must be passed.")
```

Comment:
This might confuse people who actually understand JSOC as Time and Wavelength are pkeys, we just expose them differently in the search. Not 100% on this reword but hows:
```suggestion
            raise ValueError("Time, Wavelength or an explicit PrimeKey must be specified.")
```